packages/ontario-frontend-cli/core/utils/project/readPackageJson.js

```diff
@@ -1,19 +1,22 @@
-const fs = require('fs');
+const fs = require('fs').promises;
 const path = require('path');
-const { ROOT_DIR } = require('../../config');
 
 /**
- * Synchronously reads the package.json file at the project root and parses its JSON content.
- * @returns {Object} The parsed package.json content as an object.
+ * Asynchronously reads the package.json file from the specified directory and parses its JSON content.
+ * 
+ * @param {string} dir - The directory to read the package.json from.
+ * 
+ * @returns {Promise<Object>} A promise that resolves to the parsed package.json content as an object.
+ * 
+ * @throws {Error} Throws an error if the file cannot be read or parsed.
  */
-function readPackageJson() {
-  const packageJsonPath = path.join(ROOT_DIR, 'package.json');
+async function readPackageJson(dir) {
```

Comment:
Do you think we should have a default value or would that be risky? 

```suggestion
async function readPackageJson(dir=process.cwd()) {
```

Reply:
Sounds good to me, I think that's a sensible default.